src/Settings.js

```diff
@@ -2,19 +2,20 @@ import {Engine} from "./engine.js";
 
 /* Settings.js */
 let Settings = {
-    CodeInstructionRunTime:         50,
-    MaxLogCapacity:                 50,
-    MaxPortCapacity:                50,
-    SuppressMessages:               false,
-    SuppressFactionInvites:         false,
-    SuppressTravelConfirmation:     false,
-    AutosaveInterval:               60,
-    DisableHotkeys:                 false,
-    ThemeHighlightColor:            "#ffffff",
-    ThemeFontColor:                 "#66ff33",
-    ThemeBackgroundColor:           "#000000",
-    EditorTheme:                    "Monokai",
-    EditorKeybinding:               "ace",
+    CodeInstructionRunTime:              50,
+    MaxLogCapacity:                      50,
+    MaxPortCapacity:                     50,
+    SuppressMessages:                    false,
+    SuppressFactionInvites:              false,
+    SuppressTravelConfirmation:          false,
+    SuppressBuyAugmentationConfirmation: false,
+    AutosaveInterval:                    60,
+    DisableHotkeys:                      false,
+    ThemeHighlightColor:                 "#ffffff",
+    ThemeFontColor:                      "#66ff33",
+    ThemeBackgroundColor:                "#000000",
+    EditorTheme:                         "Monokai",
+    EditorKeybinding:                    "ace",
 }
 
 function loadSettings(saveString) {
```

Comment:
We should be setting the default value here so that way returning players will get the new property.

Something like: 
`Settings.SuppressBuyAugmentationConfirmation = Settings.SuppressBuyAugmentationConfirmation || false` to default to false.